Lib/test/test_asyncore.py

```diff
@@ -431,9 +431,11 @@ def test_resource_warning(self):
     def test_close_twice(self):
         fd = os.open(support.TESTFN, os.O_RDONLY)
```

Comment:
Add something like: self.addCleanup(os.close, fd). It's enough to fix "./python -m test -R 3:3 test_asyncore".